src/ui/wheel.js

```diff
@@ -94,7 +94,7 @@ export function getOffscreenImageData(size, categories, options) {
   let canvas;
   try {
     canvas = new OffscreenCanvas(size, size);
-  } catch (e) {
+  } catch {
```

Comment:
I thought it was a recent feature, but it was introduced in ES2019, so removing unused catch parameters is quite safe.